src/isIsogram.test.js

```diff
@@ -6,4 +6,27 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`should return true 
+    if word is an isogram`, () => {
+    expect(isIsogram(''))
+      .toBeTruthy();
+  });
+
+  it(`should return true 
+    if word is empty`, () => {
+    expect(isIsogram(''))
+      .toBeTruthy();
+  });
+
+  it(`should be case-insensitive`, () => {
```

Comment:
Consider using a more descriptive test case. For example, you can test with the word 'CaseInsensitive' and expect it to be true.